presto-plugin-toolkit/src/main/java/com/facebook/presto/plugin/base/security/FileBasedAccessControl.java

```diff
@@ -74,22 +71,22 @@ public FileBasedAccessControl(FileBasedAccessControlConfig config)
     }
 
     @Override
-    public void checkCanCreateSchema(ConnectorTransactionHandle transactionHandle, Identity identity, String schemaName)
+    public void checkCanCreateSchema(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity, String schemaName)
```

Comment:
What is high level reason behind update?

Reply:
Had to bring in this change for code to build. Same change was made to all the signature of all the other functions in this file - we just needed to update our changed functions to match.